tests/vignette.js

```diff
@@ -480,7 +526,7 @@ QUnit.test('Thumbnailer recognizes vignette URLs vs static.wikia.nocookie.net/ U
 				height: 512
 			},
 			expectedOutput: 'https://static.wikia.nocookie.net/6f2431a7-19de-494c-91d0-391536b44377' +
-			'/scale-to-width/width/256'
+			'/scale-to-width/256'
```

Comment:
Yup, tests for the previous PR failed in Travis:
https://travis-ci.org/Wikia/vignette-js/builds/97197918

What's interesting is that Github shows they passed. I think we have the test script broken.